src/topp/IDPosteriorErrorProbability.cpp

```diff
@@ -221,7 +222,10 @@ class TOPPIDPosteriorErrorProbability :
       }
 
       // fit to score vector
-      bool return_value = PEP_model.fit(score.second[0]);
+      //TODO choose outlier handling based on search engine? If not set by user?
+      //XTandem is prone to accumulation at min values/censoring
```

Comment:
could this be detected by looking at the smallest non-empty bin? if that one contains e.g., the largest number of counts it is set to 0?

Reply:
or is this properly handled now?

Reply:
No, this case is still open. Unfortunately we don't work with bins here. We could look for a big number of equal scores in the low range. AFAIR it's always the same minimum score but I would need more data of many settings to see when and why xtandem actually produces such weird distributions sometimes.

Reply:
Ignoring would be a first step but IMHO censoring should be handled during fitting by e.g. taking the likelihood of being smaller than the censoring threshold (i.e. CDF) instead of a point estimate at that threshold. But I would like to see how often and how pronounced it happens and especially why (maybe it can be avoided in the search settings).

Reply:
A quick fix with probably little impact on fits in general would be to ignore all values equal or above 99.9 % percentile or equal and below 0.01.

Reply:
I gave it a little try but I have no data to test it.